### PR TITLE
Add a pending test for the datatype query fn

### PR DESCRIPTION
**Pull request by cap10morgan**
This introduces a new ^:pending test metadata that we should filter out
of regular test runs if we end up adopting it. I may introduce that in a
subsequent PR. If nothing ever comes of that, feel free to remove it.

Part of my work on #576

### CODE REVIEW

**Comment by cap10morgan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #607** <a href="https://app.graphite.dev/github/pr/fluree/db/607" target="_blank">Graphite</a> 
    * **PR #608** <a href="https://app.graphite.dev/github/pr/fluree/db/608" target="_blank">Graphite</a>  👈
      * **PR #609** <a href="https://app.graphite.dev/github/pr/fluree/db/609" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/fluree/db/608?utm_source=stack-comment).

File: test/fluree/db/query/fql_test.clj [hunk 1]
```diff
@@ -238,6 +238,30 @@
...
+                       :ex/age  36}
+                      {:id      :ex/bart
+                       :ex/name "Bart"
+                       :ex/age  "forever 10"}]))]
+    (testing "including datatype in query results"
+      (let [query   '{:select [?age ?dt]
+                      :where  [[?s :ex/age ?age]
+                               {:bind {?dt (datatype ?age)}}]}
+            results @(fluree/query db query)]
+        (is (= [["forever 10" 1] [36 8]]
```
Comment (mpoffald) [inline test/fluree/db/query/fql_test.clj:260]: When the datatype filtering is implemented, will the datatypes returned by queries be iris?

Comment (cap10morgan) [reply test/fluree/db/query/fql_test.clj:260]: Yeah that's the plan. I can update the test in here to expect that.

**Comment by cap10morgan** [comment]
### Merge activity

* **Oct 27, 1:36 PM**: @cap10morgan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/fluree/db/608).

**Pull request merged**